lib/sipper/runner.ex

```diff
@@ -14,6 +16,19 @@ defmodule Sipper.Runner do
     Sipper.FeedParser.parse(feed)
   end
 
+  defp ignore_episodes(episodes, ignore_episodes) do
+    Enum.filter(episodes, fn {title, _} ->
+      Enum.all?(ignore_episodes, fn x -> !String.contains?(title, x) end)
+    end)
+  end
+
+  defp change_download_order(episodes, ascending) do
+    cond do
+      ascending -> Enum.reverse(episodes)
+      true -> episodes
+    end
+  end
```

Comment:
How do you feel about pattern matching on method signatures instead here? E.g. `defp change_download_order(episodes, true), do: …`

Would let us get rid of the `cond`.